evil-collection-git-timemachine.el

```diff
@@ -0,0 +1,22 @@
+;;; evil-collection-git-timemachine.el --- Bindings for `git-timemachine' -*- lexical-binding: t -*-
+;; Author: William Carroll <wpcarro@gmail.com>
+
+;;; Commentary:
+;; Evil keybindings for `git-timemachine' that conform to the principles outlines in evil-collection
+
+;;; Code:
+(require 'evil)
+(require 'git-timemachine)
```

Comment:
This should be

```
(require 'git-timemachine nil t)
```

for compilation purposes.

Reply:
@jojojames I'll take your word for it! I'm less familiar with these requirements. I found prior art in other modules in this repo that use `(require 'blah)`, but I presume what you're proposing is more defensive.

Reply:
Yeah those are when the package is built-in.